src/modules/dashboard/components/Dialogs/EditExpenditureDialog/EditExpenditureDialogForm.tsx

```diff
@@ -32,8 +32,12 @@ export const MSG = defineMessages({
     id: `dashboard.EditExpenditureDialog.EditExpenditureDialogForm.descriptionText`,
     defaultMessage: `This Payment is currently Locked. Either a Motion, or a member with the Arbitration permission are required to make changes.`,
   },
+  note: {
+    id: `dashboard.EditExpenditureDialog.EditExpenditureDialogForm.note`,
+    defaultMessage: `Note: if this variable has previously been voted on, subsequent votes will require more Reputation to be in favour in order for changes to be valid.`,
```

Comment:
Can we please remove this paragraph, then just have a text link in the above paragraph with the text 'Learn more'. Like in this design - https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=7057%3A334987

The link will then go to this URL: https://colony.gitbook.io/colony/expenditures/advanced-payments#editing-locked-payments